### PR TITLE
[CUBRIDQA-140] generate call stack and mark flag for case when core d…

### PR DESCRIPTION
**Pull request by supler**
…ump is generated

I will test it

### CODE REVIEW

**Comment by supler** [comment]
finish testing, once it's reviewed, I will compile packages to commit.

File: CTP/sql/src/com/navercorp/cubridqa/cqt/console/util/CommonFileUtile.java [hunk 1]
```diff
@@ -500,6 +503,46 @@ public static boolean isLinux() {
...
+	
+	private static void searchCoreFromChildenDirectory(File file, List<File> list, List<String> existsCoreList){
+		if (file.exists() && file.isDirectory()) {
+			File[] listFiles = file.listFiles();
+			for (File child : listFiles) {
+				if (child.isDirectory()) {
+					searchCoreFromChildenDirectory(child, list, existsCoreList);
+				}else{
+					String fileName = child.getName();
+					String suffer = fileName.substring(fileName.indexOf(".") + 1, fileName.length());
```
Comment (fanzaiqiang) [inline CTP/sql/src/com/navercorp/cubridqa/cqt/console/util/CommonFileUtile.java]: typo:
suffer -> suffix

File: CTP/sql/src/com/navercorp/cubridqa/cqt/console/util/CommonFileUtile.java [hunk 2]
```diff
@@ -500,6 +503,46 @@ public static boolean isLinux() {
 		return list;
 	}
 
+	public static List<File> getCoreFiles(String targetDirectory, List<String> existsCoreList){
+		List<File> list = new ArrayList<File>();
+		File src = new File(targetDirectory);
+		if (src.exists() && src.isDirectory()) {
+			searchCoreFromChildenDirectory(src, list, existsCoreList);
+		}else{
+			String fileName = src.getName();
```
Comment (fanzaiqiang) [inline CTP/sql/src/com/navercorp/cubridqa/cqt/console/util/CommonFileUtile.java]: It's better to remove duplicated codes.

File: CTP/sql/src/com/navercorp/cubridqa/cqt/console/bo/ConsoleBO.java [hunk 3]
```diff
@@ -638,6 +646,29 @@ private synchronized void saveResults(String caseFile) {
 		}
 	}
 
+	public void saveCoreCallStackFile(String caseFile, List<File> coreFileList){
```
Comment (fanzaiqiang) [inline CTP/sql/src/com/navercorp/cubridqa/cqt/console/bo/ConsoleBO.java]:
```
SUMMARY:
core.1111 [cub_cas]  <summary>
core.2222 [cub_server] <summary>

================== core.1111 ==================
......
================== core.2222 ==================
......

```

**Pull request merged**